src/__snapshots__/geoclient.test.js.snap

```diff
@@ -1360,87 +1166,89 @@ New York, NY 10007",
       Object {
         "address_components": Array [
           Object {
-            "long_name": "254",
-            "short_name": "254",
+            "long_name": "55",
+            "short_name": "55",
             "types": Array [
-              "street_number",
+              "premise",
             ],
           },
           Object {
-            "long_name": "Broadway",
-            "short_name": "Broadway",
+            "long_name": "Financial District",
+            "short_name": "Financial District",
             "types": Array [
-              "route",
+              "neighborhood",
+              "political",
             ],
           },
           Object {
-            "long_name": "Manhattan",
-            "short_name": "Manhattan",
+            "long_name": "Vivekanand Puri",
+            "short_name": "Vivekanand Puri",
             "types": Array [
               "political",
               "sublocality",
-              "sublocality_level_1",
+              "sublocality_level_2",
             ],
           },
           Object {
-            "long_name": "New York",
-            "short_name": "New York",
+            "long_name": "Sarai Rohilla",
+            "short_name": "Sarai Rohilla",
             "types": Array [
-              "locality",
               "political",
+              "sublocality",
+              "sublocality_level_1",
             ],
           },
           Object {
-            "long_name": "New York County",
-            "short_name": "New York County",
+            "long_name": "Delhi",
+            "short_name": "Delhi",
             "types": Array [
-              "administrative_area_level_2",
+              "locality",
               "political",
             ],
           },
           Object {
-            "long_name": "New York",
-            "short_name": "NY",
+            "long_name": "Delhi",
+            "short_name": "Delhi",
             "types": Array [
               "administrative_area_level_1",
               "political",
             ],
           },
           Object {
-            "long_name": "United States",
-            "short_name": "US",
+            "long_name": "",
+            "short_name": "IN",
             "types": Array [
               "country",
               "political",
             ],
           },
           Object {
-            "long_name": "10007",
-            "short_name": "10007",
+            "long_name": "110007",
+            "short_name": "110007",
             "types": Array [
               "postal_code",
             ],
           },
         ],
-        "formatted_address": "254 Broadway, New York, NY 10007, USA",
+        "formatted_address": "55, Financial District, Vivekanand Puri, Sarai Rohilla, Delhi, 110007, India",
```

Comment:
This is a strange change, might need to revisit the intent of this test and update it accordingly.